vertx-mqtt-it/src/main/java/io/vertx/openshift/mqtt/MqttBroker.java

```diff
@@ -0,0 +1,140 @@
+package io.vertx.openshift.mqtt;
+
+import io.netty.handler.codec.mqtt.MqttQoS;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.net.PemKeyCertOptions;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.mqtt.MqttEndpoint;
+import io.vertx.mqtt.MqttServer;
+import io.vertx.mqtt.MqttServerOptions;
+import io.vertx.mqtt.MqttTopicSubscription;
+
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Martin Spisiak (mspisiak@redhat.com) on 15/03/18.
+ */
+public class MqttBroker extends AbstractVerticle {
+
+  @Override
+  public void start() {
+    MqttServerOptions options = new MqttServerOptions()
+      .setKeyCertOptions(new PemKeyCertOptions()
+        .setKeyPath("private.pem")
+        .setCertPath("public.pem"))
+      .setSsl(true)
+      .setPort(1885);
+
+    MqttServer mqttServer = MqttServer.create(vertx, options)
```

Comment:
Any specific reasons for saving this instance ? The following insecure instance is not saved.